index.html

```diff
@@ -20,10 +20,306 @@
+                    </li>
+                  </ul>
+                </div> 
+                <!--Button to sign into/sign up for Netflix-->
+                <div class="watch--bottom"> 
+                    <button class="white--button">View all</button>
+                </div>
+            </div>
+        </section>
+      <!--End of content section: watch-->
+
+    <!--Content Section: Behind the Scenes-->    
+        <section>
+          <h2 class="sr-only">Behind the scenes section</h2>
+            <div class="bts-section">
+              <div class="bts-text">
+                  <h2 class="bts--title">Go behind the scenes</h2>
+                  <p class="subtitle">Learn about the team, crew, actors and all their hard work that brought Wednesday to death</p>
+                  <div class="center--button--alignment">
+                    <button class="white--button">See more</button>
+                  </div>
+              </div>
+            </div>
+        </section>
+    <!--End of Content Section: Behind the Scenes-->
+
+    <!--Content Section: Quizzes-->
+        <section>
+          <h2 class="sr-only">Wednesday related quiz section</h2>
+            <div class="quizzes-section">
+              <div class="quizzes-text">
+                <h2 class="quiz-title">Which character are you?</h2>
+                <p class="subtitle">Find out this and more with our quizzes</p>
+                <div class="center--button--alignment">
+                  <button class="white--button">Go to Quizzes</button>
+                </div>
+              </div>
+            </div>
+        </section>
+    <!--End of Content Section: Quizzes-->
+    
+    <!--Content Section: Store--> 
+      <section>
+        <h2 class="sr-only">Store</h2>
+          <div class="store">
+            <h2 class="Shop--title">Visit our woeful shop</h2>
+            <!--Store Cards. Separate unordered lists that contatin the product image and product name-->
+            <div class="cards--section">
+              <ul>
+                <li>
+                  <img
+                  class="card--image"
+                  src="./images/sweater.png"
+                  alt="Sweater with Wendesday on it">
+                </li>
+                <li class="card--text">Clothes</li>
+              </ul>
+              <ul>
+                <li>
+                  <img
+                  class="card--image"
+                  src="./images/Bag.png"
+                  alt="Hand bag that says what would Wednesday do">
+                </li>
+                <li class="card--text">Accessories</li>
+              </ul>
+              <ul>
+                <li>
+                  <img
+                  class="card--image"
+                  src="./images/Wednesday-Pop.png"
+                  alt="Wednesday Addams funko pop">
+                </li>
+                <li class="card--text">Collectibles</li>
+              </ul>
```

Comment:
You have three separate `<ul>..</ul>` lists here each with one or two list items.  

I would recommend merging them into one `<ul>` list. 

```html
<ul>
  <li>..you could have a nested list here or use divs to organize your content...</li>
  <li>..you could have a nested list here or use divs to organize your content...</li>
  <li>..you could have a nested list here or use divs to organize your content...</li>
</ul>
```